lib/src/inline_parser.dart

```diff
@@ -274,6 +274,118 @@ class AutolinkSyntax extends InlineSyntax {
   }
 }
 
+/// Matches autolinks like `http://foo.com`.
+class AutolinkExtensionSyntax extends InlineSyntax {
+  static const START = r'(?:^|[\s*_~(>])';
```

Comment:
Dart has moved to lowerSnakeCase for const stuff. Could you update these variable names?